docs/source/theory/intro.rst

```diff
@@ -3,6 +3,27 @@
 Introduction
 ============
 
+Concepts
+--------
+
+Reference Trajectory
+""""""""""""""""""""
+
+ImpactX is an *s*-based beam dynamics code, evolving particles *relative* to a reference trajectory.
+
```

Comment:
Instead of "evolving particles..." what about "tracking particles using lattice position _s_ as the independent dynamical variable.  Particle phase space coordinates are specified relative to a nominal reference trajectory.

Reply:
Thanks! Applied and I added one more concept section about the space charge kick.